src/main/java/org/recordrobotics/charger/RobotContainer.java

```diff
@@ -35,16 +38,18 @@ public class RobotContainer {
 	/** The container for the robot. Contains subsystems, OI devices, and commands. */
 	public RobotContainer() {
 		// Configure the button bindings
-		_controlInput = new DoubleControl(Constants.Control.DOUBLE_GAMEPAD_1, Constants.Control.DOUBLE_GAMEPAD_2);
+		_controlInput = new SingleControl(Constants.Control.LEGACY_GAMEPAD);
```

Comment:
why are we creating a single control object and not a double?
(side note, we need to rename the RobotMap.java variable `LEGACY_GAMEPAD` to `SINGLE_GAMEPAD`)